pkg/cmd/server/start/start_master.go

```diff
@@ -87,9 +89,9 @@ func NewCommandStartMaster() (*cobra.Command, *MasterOptions) {
 			if err := options.StartMaster(); err != nil {
 				if kerrors.IsInvalid(err) {
 					if details := err.(*kerrors.StatusError).ErrStatus.Details; details != nil {
-						fmt.Fprintf(c.Out(), "Invalid %s %s\n", details.Kind, details.ID)
+						fmt.Fprintf(out, "Invalid %s %s\n", details.Kind, details.ID)
```

Comment:
You could actually create a struct that contains the `Out` and the `GetOut` method and anonymously include it....  Future TODO perhaps?